ERCS/erc-7802.md

```diff
@@ -0,0 +1,259 @@
+
+`IERC7802` is intentionally lightweight, ensuring minimal overhead for implementation. Its modular design enables extensibility, allowing additional features—such as mint/burn limits, transfer fees, or bridge-specific access control mechanisms—to be layered on top without modifying the base interface. 
+
+## Motivation
+
+All rollups and multiple important sidechains (Gnosis and Polygon PoS, among others) implement canonical bridges that embed their security into some part of the network's core architecture. These bridges do not have mint/burn rights over original tokens, so they usually lock (unlock) liquidity on the native chain and then mint (burn) a non-equivalent representation on the other. Mint/burn is used because the native token is non-existent on that side, so they must create a new representation. However, each bridge implements a different interface for minting/burning on non-native chains.
+
+This interface fragmentation is a massive issue in cross-chain communication among chains via third-party bridges or future canonical solutions. At this point, it is clear that every bridge would benefit from a standardized interface for minted/burnt tokens. 
+
+There have been different attempts in the past to standardize token-bridging interfaces. However, third-party providers are also developing cross-chain token frameworks. Each framework defines its features, like rate limits and fee switches, and implements its mint and burn versions. The resultant interfaces become highly specific, lacking naming conventions and structures.
+
+The proposed interface includes the most relevant and minimal set of actions used by most of these standards. These actions also do not require any governance or owner participation, in contrast, for instance, to set rate limits.
+
+## Specification
+
+This ERC introduces the `IERC7802` interface.
+
+### Interface Identification
+
+The interface identifier for `IERC7802` is **`0x33331994`**, calculated according to [ERC-165](./eip-165.md) as the XOR of the function selectors of the two functions in the interface:
+
+```solidity
+bytes4 constant INTERFACE_ID_IERC7802 = 
+        bytes4(keccak256("crosschainMint(address,uint256)")) ^
+        bytes4(keccak256("crosschainBurn(address,uint256)"));
+```
+
+or via Solidity as 
+
+```solidity
+type(IERC7802).interfaceId
+```
+
+Implementors MUST ensure that the `supportsInterface` method of ERC-165 returns true for this interface ID to indicate support for `IERC7802`.
+
+
+### Methods
+
+**`crosschainMint`**
+
+Mints `_amount` of token to address `_account`. 
+
+This function works as the minting entry point for bridge contracts. Each implementation is responsible for its access control logic.
```

Comment:
You can move the access control notes on each method to the security considerations section.